tensorflow/compiler/jit/xla_compilation_cache.h

```diff
@@ -189,6 +214,30 @@ class XlaCompilationCache : public ResourceBase {
   absl::flat_hash_map<string, ClusterCompileStats> cluster_compile_stats_
       TF_GUARDED_BY(cluster_compile_stats_mu_);
 
+  struct AsyncCompilation {
```

Comment:
Might be better to call this `AsyncCompilationState`.

Reply:
done